pandas/core/generic.py

```diff
@@ -6888,29 +6888,33 @@ def tshift(self, periods=1, freq=None, axis=0):
 
     def truncate(self, before=None, after=None, axis=None, copy=True):
         """
-        Truncates a sorted DataFrame/Series before and/or after some
-        particular index value. If the axis contains only datetime values,
+        Truncate a DataFrame/Series before/after some index value.
+
+        If the axis contains only datetime values,
         before/after parameters are converted to datetime values.
 
         Parameters
         ----------
         before : date, string, int
-            Truncate all rows before this index value
+            Truncate all rows before this index value.
         after : date, string, int
-            Truncate all rows after this index value
+            Truncate all rows after this index value.
         axis : {0 or 'index', 1 or 'columns'}
-
-            * 0 or 'index': apply truncation to rows
-            * 1 or 'columns': apply truncation to columns
-
             Default is stat axis for given data type (0 for Series and
-            DataFrames, 1 for Panels)
+            DataFrames).
         copy : boolean, default is True,
-            return a copy of the truncated section
+            Return a copy of the truncated section.
 
         Returns
         -------
-        truncated : type of caller
+        type of caller
+            The truncated DataFrame/Series.
+
+        See Also
+        --------
+        DataFrame.truncate : Truncate a DataFrame before/after some index
```

Comment:
we don't have these defined in Series/DataFrame, so not sure these can be here @jorisvandenbossche

Reply:
Yes, since the docstring is shared for both Series and DataFrame, those "see also" are referring to themselves. So indeed not needed in this case (and you ignore the error in the validation script about missing "See Also")